### PR TITLE
Created CentOS 6.5 min and max payload installs

### PR DESCRIPTION
**Pull request by geoff-reid**
@keedya @tannoa2 @uppalk1 @RackHD/corecommitters

### CODE REVIEW

File: example/samples/install_centos_6_payload_full.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                    "password": "123456",
+                    "uid": 1011,
+                    "sshKey": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDJQ631/sw3D40h/6JfA+PFVy5Ofz6eu7caxbv0zdw4fTJsrFcOliHZTEtAvqx7Oa8gqSC6d7v61M0croQxtt1DGUcH2G4yhfdZOlK4pr5McwtX0T/APACdAr1HtP7Bt7u43mgHpUG4bHGc+NoY7cWCISkxl4apkYWbvcoJy/5bQn0uRgLuHUNXxK/XuLT5vG76xxY+1xRa5+OIoJ6l78nglNGrj2V+jH3+9yZxI43S9I3NOCl4BvX5Cp3CFMHyt80gk2yM1BJpQZZ4GHewkI/XOIFPU3rR5+toEYXHz7kzykZsqt1PtbaTwG3TX9GJI4C7aWyH9H+9Bt76vH/pLBIn rackhd@rackhd-demo"
+                }
+            ],
+            "rootSshKey": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDJQ631/sw3D40h/6JfA+PFVy5Ofz6eu7caxbv0zdw4fTJsrFcOliHZTEtAvqx7Oa8gqSC6d7v61M0croQxtt1DGUcH2G4yhfdZOlK4pr5McwtX0T/APACdAr1HtP7Bt7u43mgHpUG4bHGc+NoY7cWCISkxl4apkYWbvcoJy/5bQn0uRgLuHUNXxK/XuLT5vG76xxY+1xRa5+OIoJ6l78nglNGrj2V+jH3+9yZxI43S9I3NOCl4BvX5Cp3CFMHyt80gk2yM1BJpQZZ4GHewkI/XOIFPU3rR5+toEYXHz7kzykZsqt1PtbaTwG3TX9GJI4C7aWyH9H+9Bt76vH/pLBIn rackhd@rackhd-demo",
+            "dnsServers": ["172.12.88.91", "192.168.20.77"],
+            "networkDevices": [
+                {
+                    "device": "eth0",
```
Comment (lanchongyizu) [inline example/samples/install_centos_6_payload_full.json]: Would names like eno16777984 be more fit for CentOS?

Comment (geoff-reid) [inline example/samples/install_centos_6_payload_full.json]: This is for CentOS 6 which still uses the old style eth<n> device names.  Starting with CentOS 7 the new predictable interface device names are used.  You can see these in the install_centos_7_payload_full.json file.

File: example/samples/install_centos_6_payload_full.json [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                    },
+                    "ipv6": {
+                        "ipAddr": "fec0::6ab4:0:5efe:157.60.14.21",
+                        "gateway": "fe80::5efe:131.107.25.1",
+                        "netmask": "ffff.ffff.ffff.ffff.0.0.0.0",
+                        "vlanIds": [101, 106]
+                    }
+                },
+                {
+                    "device": "eth4",
```
Comment (yyscamper) [inline example/samples/install_centos_6_payload_full.json]: This name is duplicated with last one. (line#35)

File: example/samples/install_centos_7_payload_full.json [hunk 3]
```diff
@@ -47,7 +47,7 @@
                     }
                 },
                 {
-                    "device": "ens2f1",
+                    "device": "ens3f0",
```
Comment (yyscamper) [inline example/samples/install_centos_7_payload_full.json]: Why use the same interface with last one? (line#35)

File: test/tests/api/v1_1/os_install_tests.py [hunk 4]
```diff
@@ -350,14 +349,19 @@ def test_install_coreos(self, nodes=[], options=None):
     @test(enabled=True, groups=['centos-6-5-minimal-install.v1.1.test'])
     def test_install_centos_6_minimal(self):
         """ Testing CentOS 6.5 Installer Workflow """
-        self.install_centos('6.5', payloadFile='install_centos_payload_minimal.json')
+        self.install_centos('6.5', payloadFile='install_centos_6_payload_minimal.json')
+
+    @test(enabled=True, groups=['centos-6-5-full-install.v1.1.test'])
+    def test_install_centos_6_full(self, nodes=[], options=None):
+        """ Testing CentOS 7 Installer Workflow """
```
Comment (yyscamper) [inline test/tests/api/v1_1/os_install_tests.py]: Typo in comment, this is Cento 6 not 7!

**Comment by yyscamper** [comment]
:+1: after fix the duplicated interface name.

**Comment by keedya** [comment]
@yyscamper, this is not a duplicate. Please check the readme in the samples directory to understand the setup. Basically, we setup 2 subnet on the same nic using vlanid

**Comment by keedya** [comment]
@yyscamper here is the testing system configuration:
https://github.com/RackHD/RackHD/blob/master/example/samples/README.md

**Comment by yyscamper** [comment]
Thanks @keedya , I wasn't aware of such special network configuration. So the duplicated interface comment is clear now.

**Pull request merged**